cookbook/case_studies/ml_training/nlp_processing/word2vec_and_lda.py

```diff
@@ -0,0 +1,331 @@
+"""
+.. _word2vec_and_lda:
+
+Word Embeddings and Topic Modelling using Gensim
+------------------------------------------------
+
+This script will create six Flyte tasks, that will:
+
+1. Generate the sample dataset
+2. Train the word2vec model.
+3. Train the LDA model and display the words per topic
+4. Compute word similarities
+5. Compute word movers distance
+6. Reduce dimensions using tsne and generate a plot using FlyteDeck
+
+"""
+
+# %%
+# First, we need to import some libraries.
```

Comment:
Can you run `pre-commit install` in the flytesnacks repo so that black and isort are used for formatting?